### PR TITLE
Updated references around Node.js runtimes

### PR DESCRIPTION
**Pull request by rraavi**
@mrutkows

### CODE REVIEW

**Review by dgrove-oss — APPROVED** [review]
Thanks!  We missed this when we updated the downloads page for the last openwhisk-runtime-nodejs release.

File: _layouts/documentation.html [hunk 1]
```diff
@@ -924,7 +924,7 @@ <h5>Creating and Invoking NodeJS actions</h5>
           <a class="indexable" id="nodejs-runtime"></a>
           <h5>OpenWhisk Runtime for NodeJS</h5>
           <p>
-              OpenWhisk supports <strong>Node.js 10</strong>, <strong>Node.js 8</strong>, and  <strong>Node.js 6</strong> runtimes where Node.js 6 being default pick by wsk CLI and Whisk Deploy.
+              OpenWhisk supports <strong>Node.js 14</strong>, <strong>Node.js 12</strong>, <strong>Node.js 10</strong>, <strong>Node.js 8</strong> and <strong>Node.js 6</strong> runtimes where Node.js 6 being default pick by wsk CLI and Whisk Deploy.
```
Comment (dgrove-oss) [inline _layouts/documentation.html:927]:
```suggestion
              OpenWhisk supports <strong>Node.js 14</strong>, <strong>Node.js 12</strong>, <strong>Node.js 10</strong> runtimes where Node.js 10 being the default picked by wsk CLI and Whisk Deploy.
```

**Review by mrutkows — APPROVED** [review]
LGTM. Thanks for updating our links and NodeJS version dependencies

**Comment by rraavi** [comment]
Associated PR for runtimes documentation- https://github.com/apache/openwhisk-runtime-nodejs/pull/178

**Review by mrutkows — APPROVED** [review]
Thanks for going the extra mile and validating many more links and correcting/removing as able/needed.  Likely, the events (events.md) should be removed as we do not have the bandwidth to track and the events have likely shrunk to a small number directly ref. OpenWhisk.  I will open an issue...

**Pull request merged**